### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by purvasharmadev**
Added image and input components

### CODE REVIEW

File: Components/Cards/card.css [hunk 1]
```diff
@@ -4,6 +4,8 @@
     background-color: var(--BACKGROUND-COLOR);
     font-family: var(--MAIN-FONT);
     overflow: hidden; 
+    width:20rem;
+    max-width:55rem;
     /* text-align: center; */
```
Comment (zoef-yt) [inline Components/Cards/card.css]: remove this line of code

File: Components/Cards/card.css [hunk 2]
```diff
@@ -99,25 +101,41 @@
 }
 
 
+.card-flex .btn{
+    padding: 0.3rem;
```
Comment (zoef-yt) [inline Components/Cards/card.css:100]: use px for units below 1rem

File: Components/Cards/card.css [hunk 3]
```diff
@@ -99,25 +101,41 @@
...
     position: absolute;
-    width: 95%;
-    padding: 2rem;
+    height: 100%;
+    width: 100%;
     bottom: 0px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* left: 0px; */
```
Comment (zoef-yt) [inline Components/Cards/card.css]: remove commented code.

File: Components/Cards/card.css [hunk 4]
```diff
@@ -99,25 +101,41 @@
...
+
 }
 
 .card-overlay .card-text{
     color: black;
+    font-size: 1.5rem;
+    font-weight: bolder;
+    /* border: solid; */
```
Comment (zoef-yt) [inline Components/Cards/card.css:125]: remove commented code

File: Components/Input/input.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+input{
+    padding: 0.5rem;
+    margin: 0.5rem;
+    outline: none;
+    border-radius: 5px;
+    font-size: 1rem;
+    border: 2px solid var(--PRIMARY-COLOR);
+    text-align: justify;
+    color: rgba(34, 29, 29, 0.918);
+    /* width: 90%; */
```
Comment (zoef-yt) [inline Components/Input/input.css:14]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by iRohitGaur — APPROVED** [review]
nice work. Good to merge 👍🏻

**Review by srishtigautam21 — APPROVED** [review]
do the changes as suggested by zoef....other than them the code is good to merge plus RM :) :)

**Review by sudipkundu999 — APPROVED** [review]
Your deploy preview shows an oval image! Not a circular image!
If that is what you intended, then it's okay, else please look into it.
Apart from that Good Work!

**Comment by purvasharmadev** [comment]
Thankyou everyone for reviewing and approving the code.

**Pull request merged**